From 07b7c37ca5d8277ea83edfb43c014436d6fa83b5 Mon Sep 17 00:00:00 2001
From: Chris Zhong <zyw@rock-chips.com>
Date: Fri, 9 Mar 2018 17:34:09 +0800
Subject: [PATCH] arm64: rockchip_linux_config: enable LEDS_IS31FL32XX

LEDS_IS31FL32XX is a led driver IC used by rk3308 evb.

Change-Id: I77c8dcd379e72bd85bffcad5f0ec51bbaf876274
Signed-off-by: Chris Zhong <zyw@rock-chips.com>
---
 arch/arm64/configs/rockchip_linux_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index 985a7d4b0df4..6f376a9fc1e8 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -408,6 +408,7 @@ CONFIG_MMC_DW_ROCKCHIP=y
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_GPIO=y
+CONFIG_LEDS_IS31FL32XX=y
 CONFIG_RTC_CLASS=y
 CONFIG_RTC_DRV_HYM8563=y
 CONFIG_RTC_DRV_RK808=y
-- 
2.35.3

